src/js/player.js

```diff
@@ -2505,6 +2633,7 @@ class Player extends Component {
         this.techCall_('src', source.src);
       }
 
+      this.changingSrc_ = false;
```

Comment:
changingSrc should be set false after the source is actual set on the tech using the src method